src/content/contributing/index.md

```diff
@@ -1,12 +1,313 @@
 ---
 title: Contributing
 lang: en
+sidebar: true
 ---
 
-# Contributing to ethereum.org
+# Contributing to ethereum.org 🦄
 
-This page is coming soon!
+ethereum.org is an open-source project. So if you want to help improve our portal to Ethereum, here's how you can help out.
 
-In the meantime, [learn how to contribute](https://github.com/ethereum/ethereum-org-website#how-can-i-contribute).
+## Ways to contribute
 
-Looking to add a dapp or wallet to the site? Learn [how to add a product](/en/contributing/adding-products/).
+- [Work on an open issue](https://github.com/ethereum/ethereum-org-website/issues)
+  _– Work we've identified that needs doing_
+- [Join the translations program ](https://ethereum.org/en/languages/#ethereum-org-translation-program)
+  _– Help us bring ethereum.org to new languages_
+  //I think this could be it's own page under contributing eventually
```

Comment:
```suggestion
```
I agree. This is not a comment however, it's displaying on the page 🤪

Reply:
Should we just make this update (moving that content to /contributing/translating/) a part of this PR?

Reply:
As discussed, let's do this in a separate PR and plan a translations section with TY